### DISCUSSION TITLE
tintin++ will crash when #kill after #map find <>

### DISCUSSION THREAD

**Discussion by unix5** [scandum/tintin#30] [Bugs]
When in a map, no matter how many rooms the map has, If you input "#map find \<a room name or room id\>", then input "#kill" later, tintin++ will crash:
`#alias bug {`
    `#map read mymap.map;`
    `#map goto 1;`
    `#map find 5;`
    `#kill;`
`};`
`bug;`
`#KILL - ALL LISTS CLEARED.`
`DEBUG_STACK[000] [000] = mainloop()`
`──────────────────────────────────────────────────`
`FATAL ERROR trap_handler (signal 11: Segmentation fault)`
`syserr_fatal(crashed)haha@myLinux:~/tintin $`

I've tried different maps for many times and got the same result.

**Comment by scandum** [comment]
Thanks, should be fixed in the beta. https://mudhalla.net/tintin-beta.tar.gz
